### PR TITLE
Stress non-production use of node port (copied from #34)

### PR DESCRIPTION
**Pull request by rickducott**
Merges in Christian's changes from #34

### CODE REVIEW

File: gloo/docs/user_guides/node_port.md [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
+By default, microservices deployed in Kubernetes have an internal flat network that is not accessible from the outside
+of the cluster. This is true even if you use Kubernetes on a public cloud (like Amazon AWS or Google Cloud).
 
-A NodePort is a way to make kubernetes services available from outside the cluster (and potentially allow access from the internet).
+A NodePort service is a way to make Kubernetes services available from outside the cluster (and potentially allow access from the internet) by opening ports on all of the nodes in the cluster and allowing traffic to go directly to the pods running within the cluster.
 
-In this document we will review how to expose Gloo via a NodePort.
+In this document, we will review how to expose Gloo via a NodePort service. It's important to note that NodePort is *not* a recommended production setting. Node port has the following drawbacks not limited to:
+
+* You must coordinate what services use what ports so there is no conflict
```
Comment (yuval-k) [inline gloo/docs/user_guides/node_port.md]: not sure one needs to manually coordinate as usually kubernetes does it for you

Comment (christian-posta) [reply gloo/docs/user_guides/node_port.md]: If you really try to use nodeport in production, those ports are tightly controlled by firewalls and other operational mechanisms. They are opened one at a time and under tight scrutiny. They would need to be managed and explicitly exposed, not left to k8s to do. In non-prod usage,

**Comment by christian-posta** [comment]
So @rickducott I don't see my changed merged in. Is this PR still valid? Or would I need a rebase?

**Comment by rickducott** [comment]
Just waiting on an approval here. I had resubmitted your PR due to a merge conflict and I couldn’t fix your fork. If you or Yuval approve we can merge

**Comment by christian-posta** [comment]
So I messed up the merge locally and just created a new PR for this: https://github.com/solo-io/solo-docs/pull/59

Can continue discussion there?

File: gloo/docs/user_guides/node_port.md (hunk 1 above)
Comment (yuval-k) [reply gloo/docs/user_guides/node_port.md]: makes sense

**Comment by rickducott** [comment]
Now I’m #36

**Pull request closed without merging**